src/language-js/parse/babel.js

```diff
@@ -168,7 +168,12 @@ const parseEstree = createParse(
   "parse",
   appendPlugins(["jsx", "flow", "estree"])
 );
-const parseExpression = createParse("parseExpression", appendPlugins(["jsx"]));
+const parseExpression = createParse(
```

Comment:
Can we create a new parser? I think this one is used somewhere else.

Reply:
New `__js_expression_in_vue` parser?

Reply:
howa bout __ts_expression?